### PR TITLE
Doc 978 update async api document

### PR DESCRIPTION
**Pull request by GodsonLeigh**

### CODE REVIEW

**Review by JungaBunga — APPROVED** [review]

File: docs/asyncapi.yaml [hunk 1]
```diff
@@ -72,6 +88,47 @@ components:
...
+          $ref: '#/components/schemas/ProgressiveReductions1'
+        progressiveReductions2:
+          $ref: '#/components/schemas/ProgressiveReductions2'
+        progressiveReductions3:
+          $ref: '#/components/schemas/ProgressiveReductions3'
+        progressiveReductions4:
+          $ref: '#/components/schemas/ProgressiveReductions4'
+        referenceAmount:
+          $ref: '#/components/schemas/ReferenceAmount'
+        totalDelinkedpayment:
```
Comment (samplackett) [inline docs/asyncapi.yaml:128]: payment should be capitalised

File: docs/asyncapi.yaml [hunk 2]
```diff
@@ -291,6 +348,82 @@ components:
       type: number
       description: Key field for calculation Data 
       example: 1234567
+    ApplicationId:
+      type: number
+      description: Key field for calculation Data
```
Comment (samplackett) [inline docs/asyncapi.yaml:353]: should this have the same description as the above? could it be rephrased to make it clearer what the difference is?

File: docs/asyncapi.yaml [hunk 3]
```diff
@@ -291,6 +348,82 @@ components:
...
+      example: 60
+    PercentageReduction4:
+      type: number
+      description: Fourth percentage bracket for progressive reductions 
+      example: 70
+    ProgressiveReductions1:
+      type: number
+      description: Value against band1 for progressive reductions 
+      example: 11000
+      format: decimal
```
Comment (samplackett) [inline docs/asyncapi.yaml:391]: not really an issue but should probably have these in the same order as the below ones, ie

- type
- format
- description
- example

**Review by samplackett — APPROVED** [review]
tumblr_m5nc6xNeMn1rqrf25o1_500

**Pull request merged**